train_segmentation.py

```diff
@@ -784,13 +780,14 @@ def main(cfg: DictConfig) -> None:
     -------
     :param cfg: (dict) Parameters found in the yaml config file.
     """
-    # Limit of the NIR implementation TODO: Update after each version
-    if 'deeplabv3' not in cfg.model.model_name and 'IR' in read_modalities(cfg.dataset.modalities):
-        logging.info(
-            '\nThe NIR modality will be fed at first layer of model alongside other bands,'
-            '\nthe implementation of concatenation point at an intermediary layer is only available'
-            '\nfor the deeplabv3 model for now. \nMore will follow on demand.'
-        )
+    # Limit of the NIR implementation
+    # FIXME: keep this warning?
+    # if 'deeplabv3' not in cfg.model.model_name and 'IR' in read_modalities(cfg.dataset.modalities):
+    #     logging.info(
+    #         '\nThe NIR modality will be fed at first layer of model alongside other bands,'
+    #         '\nthe implementation of concatenation point at an intermediary layer is only available'
+    #         '\nfor the deeplabv3 model for now. \nMore will follow on demand.'
+    #     )
```

Comment:
This warning is broken since we wouldn't use cfg.model.model_name anymore. It's an easy fix, but not sure this warning is necessary. Our smp models can read NIR and recycle other bands' weights. Maybe some documentation could be added to our deeplav3 dualhead and that would be enough. @CharlesAuthier

Reply:
yeaah its ok with me

Reply:
Ok! I'll remove this in PR #297